app/src/main/java/org/oppia/android/app/maven/BUILD.bazel

```diff
@@ -0,0 +1,86 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_binary", "kt_jvm_library", "kt_jvm_test")
```

Comment:
This & related files should be part of the script package, e.g. ``script/src/java/org/oppia/android/script/maven``. See #3374 for an idea on how this should be set up.